namespaces/live-1.cloud-platform.service.justice.gov.uk/manage-soc-cases-dev/resources/rds.tf

```diff
@@ -0,0 +1,97 @@
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+resource "random_id" "id" {
+  byte_length = 8
+}
+
+module "dps_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.3"
```

Comment:
Hi, Thank you for the PR.
Could you please just up to the latest release for the the RDS module.
https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance/releases/tag/5.4

Reply:
Thanks for the review. That's done now & latest changes from master also merged.